renderer/features/messages/AddMessage.js

```diff
@@ -81,6 +93,14 @@ const AddMessage = props => {
 
     if (paymentAmountResponse) {
       const { amount, memo } = paymentAmountResponse;
+      const satAmount = parseInt(amount, 10);
+      if (isNaN(satAmount) || !amount.match(/^[0-9]+$/) || satAmount <= 0) {
+        queue.alert({
+          title: 'Invalid Send Amount',
+          body: `Amount must be an integer greater or equal to zero`
+        });
+        return;
+      }
       handleSendMessage({
         message: `${memo}`,
         amount: parseInt(amount, 10) * 1000,
```

Comment:
minor nit but might as well replace this with satAmount like you did in previous flow

Reply:
oh, yeah, missed that 🙈

rebased